README.md

```diff
@@ -67,13 +93,15 @@ Not all CanDIG modules are required for a minimal installation. The `CANDIG_MODU
 
 By default (if you copy the sample file from `etc/env/example.env`) the installation includes the minimal list of modules:
 
+```
   CANDIG_MODULES=minio htsget-server katsu candig-data-portal
+```
 
 Optional modules follow the `#` and include federation service, various monitoring components, workflow execution, and some older modules not generally installed.
 
 For federated installations, you will need `federation-service`.
 
-For production deployments, you will probably want to include  `federation-service weavescope logging monitoring`. Be aware that the last three require more resources, includeing storage.
+For production deployments, you will probably want to include  `federation-service`. Be aware that the last three require more resources, includeing storage.
```

Comment:
If we are taking out the three monitoring services, then we can also remove the sentence starting with "Be aware..."